### ISSUE TITLE
Project Website Drama

### ISSUE THREAD

**Issue by omnituensaeternum** [Galaxzy345/Deadcord#10]
So in response to a mod in the discord making a shitty bloated website on a free domain, I figured I'd go out of my way and do it properly, buying deadcord.com and setting everything up with cloudflare and ssl, serving it off of my VPS, etc. Also even going as far to sit down and Rewrite the shitty sites code and make it half decent. (the only part of which I kept was the glowing animation which was likely generated on [animista](https://animista.net/) ) Then your mods decided I skidded the site because it looked very similar, which kinda was the point. Had they bothered to read the code they would know that 95% of it was my own and I even credited the mod who made the first site in the html code with a comment, I had even planned on updating it later to say the site was maintained by both me and said mod since he made the first one. But ofc, he instead decided to demand I take it down, then ban me when I refused to. I, out of generosity, paid for a proper domain with my own money. Then put time and effort into making a well written clean & simple website for the project. Only to get berated, and threatened to take it down because it looked similar to the first one which was poorly written and unprofessional. Also I was eventually banned for refusing to take it down, last I checked there were no rules that applied to that situation.

I'll let the people who find this project and those who follow it decide for themselves, the original site (if it remains up) can be found at [deadcord.ga](deadcord.ga)  and mine can be found at [deadcord.com](https://www.deadcord.com)

I wouldn't have made this issue had they taken a more polite approach to the situation. 

TL;DR Deadcord discord mod made a shitty unprofessional website with a free domain, I came along, spent my own money and time doing it the right way, only to get unrightfully banned from the discord because I refused to take it down.

**Comment by Galaxzy345** [comment]
Thanks for letting me know.
